### PR TITLE
finally add changelog for v0.3.11

### PR DESCRIPTION
**Pull request by whyrusleeping**
License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

**Comment by whyrusleeping** [comment]
@RichardLitt wanna help me look this over?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,47 @@
...
+This is the final ipfs version before the transition to v0.4.0.
+It introduces a few stability improvements, bugfixes, and increased
+test coverage.
+
+* Features
+  * add 'get' and 'patch' to the allowed gateway commands (@whyrusleeping)
+
+* BugFixes
+  * Fix path parsing for add command (@djdv)
+  * namesys: Make paths with multiple segemnts work. Fixes #2059 (@Kubuxu)
```
Comment (rht) [inline CHANGELOG.md]: s/segemnts/segments/

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,47 @@
 * Incompatible Changes
   * the default for '--type' in 'ipfs pin ls' is now "all" (@chriscool)
 
+### 0.3.11 - 2016-01-12
+
+This is the final ipfs version before the transition to v0.4.0.
```
Comment (RichardLitt) [inline CHANGELOG.md]: Capitalise IPFS

**Comment by RichardLitt** [comment]
Fixed. LGTM now.

**Comment by rht** [comment]
@RichardLitt if you look at the past commits in CHANGELOG.md, not all of them are capitalized. The option is either to keep it consistent with the commit messages, or fix it once for all.

**Comment by Dignifiedquire** [comment]
There is no note about the webui update

**Comment by RichardLitt** [comment]
@rht I don't think we have to make a hard & fast rule here about that. There was at least a spelling error in one of the commits which needed to be fixed. @whyrusleeping asked me to look over it; this is what I naturally do when I look over things. :)

**Comment by RichardLitt** [comment]
@Dignifiedquire Should that be in the go-ipfs changelog?

**Comment by Dignifiedquire** [comment]
@RichardLitt I think so, as it changes the version that is being shipped with the binary (and it was a commit into this repo)

**Comment by rht** [comment]
webui update came after 0.3.11: https://github.com/ipfs/go-ipfs/commits/master?page=6.

**Comment by rht** [comment]
(I wasn't being asked, but just commented of imo on the consistency)

**Comment by Dignifiedquire** [comment]
> webui update came after 0.3.11: https://github.com/ipfs/go-ipfs/commits/master?page=6.

:scream:

**Comment by whyrusleeping** [comment]
the webui update is technically part of 0.3.11 (its inside the tagged release)

**Comment by rht** [comment]
Also, there is http://manpages.ubuntu.com/manpages/lucid/man1/git-dch.1.html for auto-generating debian changelog, e.g. http://changelogs.debian.net/coreutils (using this would pick the "no strict capitalizing of the commit message" option).

**Comment by whyrusleeping** [comment]
`git log --oneline v0.3.10..v0.3.11 | grep -v Merge > full list` and then i organize, and remove redundant messages

**Comment by whyrusleeping** [comment]
@RichardLitt @dignifiedquire do i have a LGTM ?

**Comment by dignifiedquire** [comment]
LGTM

**Comment by rht** [comment]
(Except for the cap consistency issue I pointed out, LRGTM)

**Comment by RichardLitt** [comment]
LGTM

**Pull request merged**